doc/source/whatsnew/v0.26.0.rst

```diff
@@ -0,0 +1,201 @@
+.. _whatsnew_0260:
+
```

Comment:
this was removed, pls merge master (and remove this file from your push)

Reply:
Sorry I'm confused.  I was fixing some issues with the v0.26.0.rst file and the index.rst file this morning.  Are you saying you do not want a v0.26.0.rst file now?

Reply:
no merge master and put the note in 1.0 exists

Reply:
Ahh, sorry I misinterpreted "use the 1.0 file" to mean "use the 1.0 file as a template for the v26 file".  I'm on it,  building doc locally now.